includes/definitions.inc.php

```diff
@@ -520,6 +520,26 @@
 $config['os'][$os]['over'][0]['text']  = 'Device Traffic';
 $config['os'][$os]['over'][1]['graph'] = 'device_processor';
 $config['os'][$os]['over'][1]['text']  = 'CPU Usage';
+$config['os'][$os]['poller_modules']['aruba-controller']       = 0;
+$config['os'][$os]['poller_modules']['bgp-peers']              = 0;
```

Comment:
The asa can run bgp, I'm not sure if they respond back using our current bgp code but if it does (or you aren't sure) then this should be removed.